tasks/create.yml

```diff
@@ -1,3 +1,13 @@
+- name: Look for the configured cloud to be returned
+  os_client_config:
+    clouds:
+      - "{{ molecule_openstack_ci_cloud }}"
+  failed_when: (openstack.clouds | length) != 1
+
+- name: Set the cloud fact
+  set_fact:
+    _cloud: "{{ openstack.clouds[0] }}"
```

Comment:
I'm confused about this. `os_client_config` returns a list of clouds even when the cloud is requested by name?

Reply:
Should've just read the docs first...

https://docs.ansible.com/ansible/latest/modules/os_client_config_module.html

It takes a list and returns a list in the clouds fact. Cool. Ignore me. :)

Reply:
*clicks "ignore @seandst" button*